### ISSUE TITLE
Transfer Learning

### ISSUE THREAD

**Issue by egrassl** [kenziyuliu/MS-G3D#22]
Hello,

I am working on a dataset similar to NTU120 and I would like to pre-train MS-G3D on NTU120 and fine-tune it on my custom dataset with only 25 classes (but same joint/bone structure).

Is there a way I can do that just editing the config files or do I need to alter some code?

**Comment by kenziyuliu** [comment]
Hi @egrassl,

Thanks for your interest! I think the repo by itself does not support transfer learning and you'll have to modify some code. However, there could be many ways to do it, and one simple, hacky way could be writing a new config file, loading pre-trained MS-G3D nets, modifying the network architecture to add/remove FC layers (`model/`), and disabling gradients to previous layers in `main.py`.

Hope this helps!
